skxray/cdi.py

```diff
@@ -96,6 +98,27 @@ def gauss(dims, sigma):
     return y / np.sum(y)
 
 
+def _fft_helper(x, option='fft'):
```

Comment:
If these are for internal use only I would just do

```python
_fft_normed = lambda x: np.fft.ifftshift(np.fft.fftn(np.fft.fftshift(x)))
_ifft_normed = lambda x: np.fft.ifftshift(np.fft.ifftn(np.fft.fftshift(x)))
```

Wouldn't it be simpler to just leave the fft in the natural order and only shift it when needed?